Services/CO.CDP.Organisation.WebApi.Tests/UseCase/UpdateOrganisationUseCaseTest.cs

```diff
@@ -628,6 +700,37 @@ await act.Should()
             Roles = [PartyRole.Buyer]
         };
 
+    private Persistence.Organisation AnotherOrganisation =>
```

Comment:
Can we name this so it reflects what's different/important about this organisation?

Reply:
It is just AnotherOrganisation, nothing specific. when user try to add additional identifiers with already existing identifier number then it creates issue.

Reply:
That's why I prefer factory methods. What are you going to when a third variant of organisation is needed?